apps/src/componentLibrary/common/constants.ts

```diff
@@ -16,3 +16,8 @@ export const componentSizeToBodyTextSizeMap: {
   s: 'body-three',
   xs: 'body-four',
 };
+
+export const dropdownColors: {[key in DropdownColor]: DropdownColor} = {
```

Comment:
minor: do we need the constants in addition to the type?

Reply:
Good question, this logic is taken from current `Button` component. I find it usefull since it eliminates possibility of typos, plus for js/jsx (not ts files) it's way more safe to use constant over filling in the strings manually without comprehensive typing/linter support.

Can delete in a follow up pr if needed though